obsplus/events/json.py

```diff
@@ -6,136 +6,44 @@
 from typing import Union
 
 import obspy
-from obspy.core.event import ResourceIdentifier, QuantityError
+from obspy.core.event import Event, Catalog
 
-from obsplus.utils.events import obj_to_dict, make_class_map
-from obsplus.constants import JSON_KEYS_TO_POP
+import obsplus.events.schema as event_schema
 
 JSON_SERIALIZER_VERSION = "0.0.0"  # increment when serialization changes
 
 
 # -------------------- events to json functions
 
 
-def cat_to_json(catalog: obspy.Catalog) -> str:
+def cat_to_json(events: Union[Catalog, Event]) -> str:
     """
-    Return a json string of the catalog object
+    Convert events (catalog or event) to json string.
     """
-    out_dict = cat_to_dict(catalog)
-    return json.dumps(out_dict)
+    if isinstance(events, Event):  # A single event was passed
+        events = Catalog(events=[events])
+    elif not isinstance(events, Catalog):  # sequence was passed
+        events = Catalog(events=events)
+    json_str = event_schema.Catalog.from_orm(events).json()
+    return json_str
 
 
-def cat_to_dict(obj):
+def cat_to_dict(events: Union[Catalog, Event]) -> dict:
```

Comment:
Nice change in this function - I'm pretty sure I wouldn't have used an `obj` kwarg, but if you go with the `catalog` kwarg above, can you use `catalog` here as well?